firmware/firmware.cpp

```diff
@@ -614,6 +614,10 @@ static const uint8_t *flash_to_tmp(const std::string &filename, uint32_t &size)
   return (const uint8_t *)(qspi_flash_address + flash_offset);
 }
 
+static bool blit_is_launcher_running() {
+  return launcher_offset == persist.last_game_offset;
+}
+
```

Comment:
Oh, entirely forgot that the firmware had `launcher_offset`... much shorter than my version, which was:
```c++
bool blit_is_launcher_running() {
  auto meta = blit_get_running_game_metadata();

  if(memcmp(meta + 1, "BLITTYPE", 8) == 0) {
    auto type_meta = reinterpret_cast<RawTypeMetadata *>(reinterpret_cast<char *>(meta) + sizeof(*meta) + 8);

    return strcmp(type_meta->category, "launcher") == 0;
  }

  return false;
}
```